File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/WriteResponse.java
package io.github.thunkier.thunkmetrc.wrapper.models;import java.util.List;
import com.fasterxml.jackson.annotation.JsonProperty;public class WriteResponse {
    @JsonProperty("Ids")
    public List<Integer> ids;
    @JsonProperty("Warnings")
    public String warnings;
}

File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/WriteResponse.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class WriteResponse(
    @JsonProperty("Ids")
    val ids: List<Int>? = null,
    @JsonProperty("Warnings")
    val warnings: String? = null
)
